tests/test_sites.py

```diff
@@ -22,16 +22,18 @@ def __exit__(self, *args):
         settings.SITE_ID = self.site_id_stable
 
 
-class SitePageTest(TestCase):
+class SiteIDTest(TestCase):
 
     def test_page_has_defaualt_site_attribute(self):
-        page = QuestionPage.objects.create()
+        with TempSiteID(1):
+            page = QuestionPage.objects.create()
         self.assertEqual(page.site.domain, 'example.com')
 
     def test_on_site_respects_SITE_ID_setting(self):
         site_1_pages = 3
         site_2_pages = site_1_pages + 1
-        site_2 = Site.objects.create()
+        with TempSiteID(1):
+            site_2 = Site.objects.create()
```

Comment:
I found another issue in this PR anyways (from working on this https://github.com/SexualHealthInnovations/callisto-core/commit/beec2e70a36c8fdfe924b590d68b95c709e50caa)